mlflow/genai/judges/constants.py

```diff
@@ -1 +1,7 @@
 _DATABRICKS_DEFAULT_JUDGE_MODEL = "databricks"
+
+# Common affirmative values that should map to YES
+_AFFIRMATIVE_VALUES = frozenset(["true", "pass", "passed", "correct", "success", "1", "1.0"])
+
+# Common negative values that should map to NO
+_NEGATIVE_VALUES = frozenset(["false", "fail", "failed", "incorrect", "failure", "0", "0.0"])
```

Comment:
Can we use GenAI to augment these lists further?

Reply:
Will try with a few different providers models